j2me/form-entry/src/org/javarosa/utilities/media/MediaUtils.java

```diff
@@ -136,13 +140,47 @@ private static String getFileFormat(String fpath){
             if(fpath.indexOf(".au") > -1) return "audio/basic";
             throw new RuntimeException("COULDN'T FIND FILE FORMAT");
         }
-
+        
+        /**
+         * Log that a media file was started, paused, or stopped (i.e., finished playing).
+         * 
+         * TODO: move this to some sort of 'form entry diagnostics' framework instead of bloating the logs.
+         * @param event From PlayerListener
+         * @param uri Media file location
+         */
+        public static void logEvent(String event, String uri) {
+            String action = "";
+            if (event == PlayerListener.STARTED) {
+                action = "start";
+            }
+            if (event == PlayerListener.STOPPED) {
+                action = "pause";
+            }
+            else if (event == PlayerListener.END_OF_MEDIA) {
+                action = "stop";
+            }
+            if (!"".equals(action)) {
+                try {
+                    Vector<String> pieces = DateUtils.split(uri, "/", false);
+                    uri = pieces.lastElement();
+                } catch (Exception e) {
+                    // just use the full URI
+                }
+                Logger.log("media", action + " " + uri);
```

Comment:
As much as I hate magic strings, it might make the data team's job a bit easier if we delimit this with something that is unlikely to exist on either side (It's possible and not super uncommon for the URI's to have spaces in them, I think)